templates/registration/register.html

```diff
@@ -2,21 +2,28 @@
 {% load staticfiles %}
 
 {% block content %}
-    <h2 class="extra-padding">Register</h2>
-    <div class="break">
-        <hr>
-    </div>
-    <div class="center">
-        <p>Sign up with one click using Facebook or Google</p>
-        <a href="{% url 'social:begin' backend='facebook-register' %}"><img class="oauth-icon" src="{% static "img/facebook-signup.png" %}" alt="Register with Facebook" /></a><br/>
-        <a href="{% url 'social:begin' backend='google-oauth2-register' %}"><img class="oauth-icon" src="{% static "img/google-signup.png" %}" alt="Register with Google" /></a>
-    </div>
-    <div class="form-input">
-        <p>or create an account the old-fashioned way</p>
-        <form action="" method="POST">
-            {% csrf_token %}
-            {{ form.as_p }}
+    <div id="register-page" class="page">
+        <h2 class="extra-padding">Sign Up</h2>
+        <div class="break">
+            <hr>
+        </div>
+        <div class="center signup-form">
+            <a href="{% url 'social:begin' backend='facebook-register' %}"><img class="oauth-icon" src="{% static "img/facebook-signup.png" %}" alt="Register with Facebook" /></a><br/>
+            <a href="{% url 'social:begin' backend='google-oauth2-register' %}"><img class="oauth-icon" src="{% static "img/google-signup.png" %}" alt="Register with Google" /></a>
+            <div class="form-input sign-up">
+                <h5><a class="signup-manual" data-open="signup-modal">Sign Up Manually</a></h5>
+            </div>
+            <p class="register-copy">We will never share your information</p>
```

Comment:
Are you sure you want to promise this? Looks pretty bad a few years down the road when you want to change the privacy policy to share data with a partner. Maybe instead you should say "We do not share your information".